source/extensions/filters/http/grpc_json_transcoder/json_transcoder_filter.cc

```diff
@@ -511,6 +512,8 @@ Http::FilterHeadersStatus JsonTranscoderFilter::decodeHeaders(Http::RequestHeade
       content_type_.assign(content_type.begin(), content_type.end());
     }
 
+    auto stream_size_before = request_in_.bytesStored();
```

Comment:
can we use concrete types here? Knowing signed vs unsigned, and the exact types we are talking about will help clue the reader into potential subtle issues, or rest assured there aren't any.

Reply:
Done.